azurerm/internal/services/frontdoor/validate/validate.go

```diff
@@ -0,0 +1,172 @@
+package validate
```

Comment:
Could we just move this one up into the front door package?

Reply:
Sure, moved.

Reply:
I tried moving it but go got confused with the path.